gen3-client/g3cmd/utils.go

```diff
@@ -32,19 +33,35 @@ func GenerateUploadRequest(guid string, url string, file *os.File) (*http.Reques
 		endPointPostfix := "/user/data/upload/" + guid
 		signedURL, _, err := function.DoRequestWithSignedHeader(profile, "", endPointPostfix, "", nil)
 		if err != nil && !strings.Contains(err.Error(), "No GUID found") {
-			log.Fatalf("Upload error: %s!\n", err)
+			log.Fatalf("Upload error: %s\n", err)
 			return nil, nil, err
 		}
 		url = signedURL
 	}
 
 	fi, err := file.Stat()
 	if err != nil {
-		log.Fatal("File Stat Error")
+		log.Fatal("File stat error\n")
```

Comment:
Can we change this to print the filename too
Something like 
"File stat error for file %s, file may be missing or unreadable because of permissions?"

Reply:
updated